mesa/space.py

```diff
@@ -422,32 +423,24 @@ def move_to_empty(
         self, agent: Agent, cutoff: float = 0.998, num_agents: int | None = None
     ) -> None:
         """Moves agent to a random empty cell, vacating agent's old cell."""
-        if len(self.empties) == 0:
+        if num_agents is not None:
+            warn(
+                (
+                    "`num_agents` is being deprecated since it's no longer used "
+                    "inside the function, it shouldn't be passed as a parameter."
```

Comment:
One last comment: the comma here should be full stop or semicolon. Everything else LGTM.

Reply:
Also, you should say "`move_to_empty`" instead of "the function."